bin/create.js

```diff
@@ -364,6 +449,22 @@ function parseDate (argv, field) {
     argv[field] = Math.floor(date.valueOf() / 1000);
 }
 
+function getSecret() {
+    try {
+        return (process.env.WEBTASK_JWT_SECRET || Crypto.randomBytes(128).toString('base64'));
```

Comment:
I think it makes sense with `--share`, but you should probably be able to override it. It's validated in `coreos-tools`, this makes the given URL the only one that'll work unless you get the secret out of `wt-cli` and sign your own tokens.